tests/test_spread.py

```diff
@@ -0,0 +1,71 @@
+from pytest import fixture
+from pytest_bdd import scenario, given, when, then, parsers
+
+from beancount.core.compare import hash_entry, includes_entries
+from beancount.loader import load_string
+from beancount.parser import printer
+
+from .context import spread
```

Comment:
Did it work for you? It doesn't work for me with Python 3.7.5 and requires this change

```suggestion
from context import spread
```

Reply:
oh ok, no it didn't work for me, I thought it was some kind of problem with my own installation or something.